src/api/form/PDFForm.ts

```diff
@@ -612,6 +581,20 @@ export default class PDFForm {
    * ```
    */
   removeField(field: PDFField) {
```

Comment:
Would you please add some unit tests for this method? This isn't something that can really be verified via the integration tests and it has a fair amount of logic now.